packages/viewer/src/components/viewer/viewer.tsx

```diff
@@ -106,6 +106,19 @@ export class Viewer {
    */
   @Event() public tap!: EventEmitter<TapEventDetails>;
 
+  /**
+   * Emits an event whenever the user double taps or clicks a location in the viewer.
+   * The event includes the location of the first tap or click.
+   */
+  @Event() public doubletap!: EventEmitter<TapEventDetails>;
```

Comment:
Is it standard for these event types to be all lowercase vs camel? Camel feels more natural to me personally but I'm good with whatever is more standard.

Reply:
In JS/HTML yes, lowercase is the norm. https://developer.mozilla.org/en-US/docs/Web/Events/touchstart